src/Bandwidth.Standard.Test/Api/CallsApiTests.cs

```diff
@@ -234,6 +234,61 @@ public void GetCallStateNotFoundRequest()
             Assert.Equal("Error calling GetCallState: ", Exception.Message);
             Assert.Equal(404, Exception.ErrorCode);
         }
+        
+        /// <summary>
+        /// Test ListCalls
+        /// </summary>
+        // [Fact]
```

Comment:
is this commented out for a reason?

Reply:
Nope! Thanks for pointing that out. It should be fixed now.